CONTRIBUTING.md

```diff
@@ -260,7 +260,15 @@ To do this you can:
 
 - Lastly, in the terminal run `terraform init` again and this time terraform will pull provider version from `tf_cache` network mirror. You can confirm this by noting the `Terraform has been successfully initialized! Using mongodb/mongodbatlas Vx.x.x from the shared cache directory` message.  
 
-
+## Code and Test Best Practices
```

Comment:
Some follow up idea:

Most of the upstream repositories address that by defining "folder structure" chapter where each significant folder is documented and visualized in the tree. 

Additionally you can reference go/terraform standards so readers can quickly gain context based on their understanding of the ecosystem

Reply:
thanks! i'll add it to the JIRA ticket above.